lms/resources/lti_launch.py

```diff
@@ -179,12 +179,7 @@ def canvas_sections_supported(self):
             # Canvas course sections feature was released.
             return False
 
-        try:
```

Comment:
No need to check for application instance existence here. We'd fail much sooner and, if for reason, we found that there's no AI at this point we rather raise an exception (from the AI service) that hide the error.